code/controllers/configuration/entries/general.dm

```diff
@@ -543,3 +543,18 @@
 	min_val = 0
 
 /datum/config_entry/str_list/motd
+
+/datum/config_entry/number/urgent_ahelp_cooldown
+	default = 300
+
+/datum/config_entry/string/urgent_ahelp_message
+	default = "No message provided"
```

Comment:
You could have a more sensible default than this!

Reply:
idk what else to put, please set me in the config?